src/main/java/hudson/plugins/synergy/SynergySCM.java

```diff
@@ -120,37 +120,14 @@ private DescriptorImpl() {
 
 		@Override
 		public boolean configure(StaplerRequest request, JSONObject formData) throws FormException {
-			ccmExe = request.getParameter("synergy.ccmExe");
-			ccmUiLog = request.getParameter("synergy.ccmUiLog");
-			ccmEngLog = request.getParameter("synergy.ccmEngLog");
-			pathName = request.getParameter("synergy.pathName");
+			ccmExe = formData.getString("synergy.ccmExe");
+			ccmUiLog = formData.getString("synergy.ccmUiLog");
+			ccmEngLog = formData.getString("synergy.ccmEngLog");
+			pathName = formData.getString("synergy.pathName");

```

Comment:
In fact I am not sure when this method would even be called. Every time you save a project, its `SCM` is recreated as a `newInstance`. `Descriptor.configure` exists for the benefit of singletons, where the object identity must remain unchanged after some special global configuration page is submitted. IOW this might have been dead code to begin with.